geom/errors.go

```diff
@@ -59,11 +59,11 @@ func wrapWithGeoJSONSyntaxError(err error) error {
 }
 
 type mismatchedGeometryCollectionDimsError struct {
-	ct1, ct2 CoordinatesType
+	CT1, CT2 CoordinatesType
```

Comment:
These need to be exported, otherwise they can't be accessed from the `geom_test` package. Because `mismatchedGeometryCollectionDimsError` isn't exported (except during tests), having `CT1` and `CT2` exported doesn't make any difference for users of the library.